rafx-framework/src/visibility/entity_id.rs

```diff
@@ -1,3 +1,8 @@
+use std::hash::Hash;
+
+/// An opaque 64-bit handle used as a unique identifier for objects in the game world
+/// that are visible or otherwise relevant to the renderer's pipeline. Each `VisibilityObject`
+/// is associated with a specific `EntityId`.
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 pub struct EntityId(u64);
```

Comment:
I'd consider making this UserId or VisibilityId (can be a separate change!)

Reply:
Will defer for now.